CHANGELOG.md

```diff
@@ -48,6 +48,7 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 - webui: get volume and pool params from query instead of route [PR #1139]
 - FreeBSD packages: add missing ddl/update 2171_2192 and 2192_2210 files [PR #1147]
 - Fix director connects to client while `Connection From Director To Client` is disabled. [PR #1099]
+- dir: miscalculation when using `always incremental keep number` [PR #1159]
```

Comment:
This is a bit too broad, maybe "miscalculation when `always incremental keep number` is greater than the number of backups determined by `always incremental job retention`